### PR TITLE
Fixes spacing for version strings

### PR DESCRIPTION
**Pull request by gfrn**

### CODE REVIEW

File: pyqt-apps/siriushla/widgets/label.py [hunk 1]
```diff
@@ -104,6 +105,13 @@ def value_changed(self, new_value):
             self.setText(time)
             return
 
+        # If it is a version string, replace multiple whitespaces with a single one
+        if self._display_format_type == self.DisplayFormat.Version:
+            version = new_value[:16] + " " + new_value[16:]
```
Comment (fernandohds564) [inline pyqt-apps/siriushla/widgets/label.py:110]: isn't this too particular? Why splitting at the 16th position?

**Review by fernandohds564 — CHANGES_REQUESTED** [review]

File: pyqt-apps/siriushla/widgets/label.py (hunk 1 above)
Comment (gfrn) [reply pyqt-apps/siriushla/widgets/label.py:110]: As per BSMP specifications regarding UDCs, version strings shall be exactly 16 characters long, always. However, we can rename this `DisplayFormat` to something like "UDCVersion" as this is the standard for ARM/DSP firmware on those.

Comment (fernandohds564) [reply pyqt-apps/siriushla/widgets/label.py:110]: I think it is a good ideia to change the name of the 'DisplayFormat' option then. This widget is used in almost all of our windows and it is not a good practice to let its interface dubious.

Comment (gfrn) [reply pyqt-apps/siriushla/widgets/label.py:110]: `Version` was changed to `BSMPUDCVersion`. It is indeed a better naming option

**Review by fernandohds564 — APPROVED** [review]

**Comment by gfrn** [comment]
Oh, one more thing: I don't have write access to the repository, so I can't merge on my own

**Review by xresende — APPROVED** [review]

**Comment by fernandohds564** [comment]
> I'll give you write permission @gfrn !

done @gfrn! Please try to merge now.

**Comment by gfrn** [comment]
Everything works now, thanks!

**Pull request merged**